README.md

```diff
@@ -79,7 +79,7 @@ sudo cri-containerd -v 2 --alsologtostderr
 ```shell
 CONTAINER_RUNTIME=remote CONTAINER_RUNTIME_ENDPOINT='/var/run/cri-containerd.sock' ./hack/local-up-cluster.sh
 ```
-For a multi node cluster installer and bring up steps using ansible and kubeadm refer to [Getting started](contrib/ansible/getting-started.md) guide.
+For a multi node cluster installer and bring up steps using ansible and kubeadm refer to [Getting started](contrib/ansible/README.md) guide.
```

Comment:
also the link text .. did you want to change that to readme?

Reply:
maybe:For a multi node cluster installer and bring up steps using ansible and kubeadm refer [here](contrib/ansible/README.md).

Reply:
Will do.

Reply:
Will do.